CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
The changelog should still reflect ops changes if they are affecting the gruntfile